graphql/src/main/java/com/linecorp/armeria/server/graphql/GraphqlServiceBuilder.java

```diff
@@ -208,18 +255,47 @@ public GraphqlService build() {
         return new DefaultGraphqlService(builder.build(), dataLoaderRegistry, useBlockingTaskExecutor);
     }
 
-    private static TypeDefinitionRegistry typeDefinitionRegistry(List<File> schemaFiles) {
+    private GraphQLSchema makeSchema() {
+        final List<URL> schemaUrls = this.schemaUrls.build();
+        final List<RuntimeWiringConfigurator> runtimeWiringConfigurators =
+                this.runtimeWiringConfigurators.build();
+        final List<GraphQLTypeVisitor> typeVisitors = this.typeVisitors.build();
+
+        if (schema != null) {
+            checkState(schemaUrls.isEmpty() && runtimeWiringConfigurators.isEmpty() &&
+                       typeVisitors.isEmpty(),
+                       "Cannot add schemaUrl(or File), runtimeWiringConfigurator and " +
+                       "typeVisitor when GraphqlSchema is specified.");
+            return schema;
+        }
+
+        final TypeDefinitionRegistry registry = typeDefinitionRegistry(schemaUrls);
+        final RuntimeWiring runtimeWiring = buildRuntimeWiring(runtimeWiringConfigurators);
+        GraphQLSchema schema = new SchemaGenerator().makeExecutableSchema(registry, runtimeWiring);
+        for (GraphQLTypeVisitor typeVisitor : typeVisitors) {
+            schema = SchemaTransformer.transformSchema(schema, typeVisitor);
+        }
+        return schema;
+    }
+
+    private static TypeDefinitionRegistry typeDefinitionRegistry(List<URL> schemaURLs) {
         final TypeDefinitionRegistry registry = new TypeDefinitionRegistry();
         final SchemaParser parser = new SchemaParser();
-        if (schemaFiles.isEmpty()) {
-            schemaFiles = defaultSchemaFiles();
+        if (schemaURLs.isEmpty()) {
+            schemaURLs = defaultSchemaURLs();
```

Comment:
nit: for consistency?
```suggestion
            schemaUrls = defaultSchemaUrls();
```